class-components/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,37 @@
+import { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+
+import './pagination.css';
+
+interface PaginationProps {
+  elementsCount: number;
+  currentPage: number;
+}
+
+const Pagination: FunctionComponent<PaginationProps> = ({

```

Comment:
```suggestion
const Pagination: FC<Props> = ({
```
если ты пропсы над компонентам описываешь, то и так понятно что это его пропсы. не нужно писать PaginationProps. хватит просто Props

Reply:
это сниппет так прописывает, а я не заморачивался, но исправил